examples/images_contours_and_fields/streamplot_demo.py

```diff
@@ -0,0 +1,82 @@
+"""
+==========
+Streamplot
+==========
+A streamplot, or streamline plot, is used to display 2D vector fields. This
```

Comment:
Then one remaining argument in favor of adding a blank line would be consistency among our example scripts, as a lot of other examples in the docs seem to have one such blank line ;). But I am also fine with no blank line if SG and rST are OK!